yt_dlp/extractor/bilibili.py

```diff
@@ -633,13 +634,35 @@ def get_entries(page_data):
         return self.playlist_result(paged_list, playlist_id)
 
 
-class BilibiliSpacePlaylistIE(BilibiliSpaceBaseIE):
-    _VALID_URL = r'https?://space.bilibili\.com/(?P<mid>\d+)/channel/collectiondetail\?sid=(?P<sid>\d+)'
+class BilibiliSpaceListBaseIE(BilibiliSpaceBaseIE):
+    def _get_entries(self, page_data, bvid_path=None):
+        for bvid in traverse_obj(page_data, bvid_path) or []:
```

Comment:
I think it would be better to do it like this; reduces code duplication and chances of usage error:
```suggestion
    def _get_entries(self, page_data, bvid_keys):
        for bvid in traverse_obj(page_data, (*variadic(bvid_keys, (str, bytes, dict, set)), ..., 'bvid', {str})):
```

Reply:
```suggestion
    def _get_entries(self, page_data, bvid_keys, ending_key='bvid'):
        for bvid in traverse_obj(page_data, (*variadic(bvid_keys, (str, bytes, dict, set)), ..., ending_key, {str})):
```

Reply:
Looks good